llm4pddl/approaches/llm_open_loop_approach.py

```diff
@@ -73,40 +73,32 @@ def _create_prompt(task: Task, plan: Plan) -> str:
                 continue
             typ_str = " ".join(objs) + " - " + str(typ)
             objects_strs.append(typ_str)
+        # Create the objects string.
+        objects_str = "\n  ".join(objects_strs)
```

Comment:
this is minor but is it necesary to have `\n .join` for line 77 and 82 instead of `\n.join`? Since you are going to a new line it seems unnecessary to me to have the space and wont minify_pddl_problem() just get rid of them?